sql/core/src/test/scala/org/apache/spark/sql/SQLQuerySuite.scala

```diff
@@ -3393,6 +3393,16 @@ class SQLQuerySuite extends QueryTest with SharedSparkSession with AdaptiveSpark
       )
     }
   }
+
+  test("SPARK-30870: Fix nested column aliasing") {
```

Comment:
Can you make the test title clearer?

Reply:
+1 for @maropu 's comment. Please revise the PR title together.

Reply:
I've changed the name of the test and the PR.

Reply:
Sorry, I've changed it again. Let me now if a different name would fit better.